src/audio/voice/playback/decode_pool.cpp

```diff
@@ -0,0 +1,75 @@
+#include "decode_pool.hpp"
+#include "constants.hpp"
+
+namespace AbaddonClient::Audio::Voice::Playback {
+
+// I have no idea what this does, I just copied it from AudioManager
```

Comment:
rtp payloads can have extension headers which you have to strip off since they arent actually part of the opus packet
https://en.wikipedia.org/wiki/Real-time_Transport_Protocol#Packet_header
now that i think about it the if condition here is probably wrong. oh well thats out of scope here :^)